modules/core/remove_orphaned.py

```diff
@@ -89,14 +98,18 @@ def rem_orphaned(self):
             }
             self.config.send_notifications(attr)
             # Delete empty directories after moving orphan files
-            logger.info("Cleaning up any empty directories...")
             if not self.config.dry_run:
-                for file in orphaned_files:
-                    src = file.replace(self.root_dir, self.remote_dir)
-                    dest = os.path.join(self.orphaned_dir, file.replace(self.root_dir, ""))
-                    util.move_files(src, dest, True)
-                    orphaned_parent_path.add(os.path.dirname(file).replace(self.root_dir, self.remote_dir))
-                    for parent_path in orphaned_parent_path:
-                        util.remove_empty_directories(parent_path, "**/*")
+                with Pool(processes=cpu_count()) as pool:
+                    orphaned_parent_path = set(pool.map(move_orphan, orphaned_files))
+
+                    logger.print_line("Removing newly empty directories", self.config.loglevel)
+                    pool.starmap(util.remove_empty_directories, zip(orphaned_parent_path, repeat("**/*")))
         else:
             logger.print_line("No Orphaned Files found.", self.config.loglevel)
+
```

Comment:
```suggestion

    def cleanup_pool(self):
        self.pool.close()
        self.pool.join()
```

Function for cleaning up pool resources.